pytradfri/device.py

```diff
@@ -140,22 +143,26 @@ def raw(self):
         return self._device.raw[ATTR_DEVICE_INFO]
 
 
-class LightControl:
-    """Class to control the lights."""
+class DeviceControl:
+    """Class to control devices."""
 
     def __init__(self, device):
         self._device = device
 
+    @property
+    def raw(self):
+        """Return raw data that it represents.
+        If regular bulb=>ATTR_LIGHT_CONTROL If GU10 light=>ROOT_SWITCH."""
+        if ATTR_LIGHT_CONTROL in self._device.raw:
+            return self._device.raw[ATTR_LIGHT_CONTROL]
```

Comment:
The type should come from another func, maybe `_device_type`? This then means we only have one place we do the selection. 

I feel that in the future `device.light_control` and `device.has_light_control` will be confusing, so we may need to move this logic up to `Device`, which can then return the appropriate Control class for the type, making `DeviceControl` the super. 

Then we'd have three functions on `Device`; 
- `device_control` (I don't care what type it is, I just want to be able to switch the device on/off)
- `light_control` (I want light specific control functions)
- `switch_control` (I want switch specific functions, whatever these may be)

However, it's very frustrating that the GU10 bulb is identified as a switch, as this makes the API very confusing to a user, I'm not sure there is a neat way for us to handle this.

Reply:
@lwis thanks! I will give this some more consideration and revert. On switches, I believe they call their "switch" a "plug", so it might be put in a whole different root path.

Reply:
@lwis should we wait for the plugs to be released before we do the reshuffling?

Reply:
I think that's wise, as we'll have to do some work to support it anyway.